template.yaml

```diff
@@ -277,6 +277,30 @@ Resources:
               ArnLike:
                 kms:EncryptionContext:aws:logs:arn: !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:*
 
+  RpPublicKeyCacheTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: RpPublicKeyCache
+      AttributeDefinitions:
+        - AttributeName: clientId
+          AttributeType: S
+        - AttributeName: keyId
+          AttributeType: S
+      KeySchema:
+        - AttributeName: clientId
+          KeyType: HASH
+        - AttributeName: keyId
+          KeyType: RANGE
+      BillingMode: PAY_PER_REQUEST
+      SSESpecification:
+        SSEEnabled: true
+        KMSMasterKeyId: !GetAtt MainKmsKey.Arn
```

Comment:
I've assumed it's OK to use `MainKmsKey` for encrypting the table. Is that OK or should a separate key be defined?

Reply:
In fact, I added encryption just to appease checkov, but is it even necessary as the data in this table isn't sensitive?

Reply:
Looks like we have specific keys for tables but I think the question is more if we need encryption on this table.

Reply:
We should have it. We don't really need the extra control or visibility it gives us in this case, but we don't really lose anything and it's not worth justifying an exception